src/lang/std/sketchcombos.ts

```diff
@@ -369,192 +373,190 @@ const setAbsDistanceCreateNode =
-        return createBinaryExpressionWithUnary([
-          createSegEnd(referenceSegName, isX),
-          createLiteral(
-            roundOff(arg - (referencedSegment?.to?.[index] || 0), 2)
-          ),
-        ])
-      }
-      return createCallWrapper(
-        'lineTo',
-        isX ? [makeBinExp(0), varValBinExp] : [varValBinExp, makeBinExp(1)],
-        tag
-      )
+    const makeBinExp = (index: 0 | 1) => {
+      const arg = getArgLiteralVal(args?.[index].expr)
+      return createBinaryExpressionWithUnary([
+        createSegEnd(referenceSegName, isX),
+        createLiteral(roundOff(arg - (referencedSegment?.to?.[index] || 0), 2)),
+      ])
     }
-  }
-
-const setAngledIntersectLineForLines: TransformInfo['createNode'] =
-  ({ referenceSegName, tag, forceValueUsedInTransform }) =>
-  (args) => {
-    const valueUsedInTransform = roundOff(
-      args[1].type === 'Literal' ? Number(args[1].value) : 0,
-      2
+    return createCallWrapper(
+      'lineTo',
+      isX ? [makeBinExp(0), varValBinExp] : [varValBinExp, makeBinExp(1)],
+      tag
     )
-    const angle = args[0].type === 'Literal' ? Number(args[0].value) : 0
-    const varNamMap: { [key: number]: string } = {
-      0: 'ZERO',
-      90: 'QUARTER_TURN',
-      180: 'HALF_TURN',
-      270: 'THREE_QUARTER_TURN',
-    }
-    const angleVal = [0, 90, 180, 270].includes(angle)
-      ? createIdentifier(varNamMap[angle])
-      : createLiteral(angle)
-    return intersectCallWrapper({
-      fnName: 'angledLineThatIntersects',
-      angleVal,
-      offsetVal:
-        forceValueUsedInTransform || createLiteral(valueUsedInTransform),
-      intersectTag: createIdentifier(referenceSegName),
-      tag,
-      valueUsedInTransform,
-    })
   }
 
-const setAngledIntersectForAngledLines: TransformInfo['createNode'] =
-  ({ referenceSegName, tag, forceValueUsedInTransform, varValA }) =>
-  (args) => {
-    const valueUsedInTransform = roundOff(
-      args[1].type === 'Literal' ? Number(args[1].value) : 0,
-      2
-    )
-    // const angle = args[0].type === 'Literal' ? Number(args[0].value) : 0
-    return intersectCallWrapper({
-      fnName: 'angledLineThatIntersects',
-      angleVal: varValA,
-      offsetVal:
-        forceValueUsedInTransform || createLiteral(valueUsedInTransform),
-      intersectTag: createIdentifier(referenceSegName),
-      tag,
-      valueUsedInTransform,
-    })
+const setAngledIntersectLineForLines: CreateStdLibSketchCallExpr = ({
+  referenceSegName,
+  tag,
+  forceValueUsedInTransform,
+  rawArgs: args,
+}) => {
+  const valueUsedInTransform = roundOff(Number(args[1].expr.value) || 0, 2)
+  const angle = Number(args[0].expr.value) || 0
+  const varNamMap: { [key: number]: string } = {
+    0: 'ZERO',
+    90: 'QUARTER_TURN',
+    180: 'HALF_TURN',
+    270: 'THREE_QUARTER_TURN',
   }
+  const angleVal = [0, 90, 180, 270].includes(angle)
+    ? createIdentifier(varNamMap[angle])
+    : createLiteral(angle)
```

Comment:
I disagree, having it contained where it's used is good in this case where it's not used anywhere else.

If this function ever gets deleted `varMap` will get delete with it, if I pull it out there's a chance it will linger until some one realises it's not used anywhere.

Reply:
The linter and type checker would catch them lingering though

Reply:
Yeah I was about to more nuance into my comment

const myVar = ...

Would get caught
but

export const myVar = ...

Would not, linter is like, "maybe it's used elsewhere"